srcpkgs/linux5.10/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'linux5.10'
 pkgname=linux5.10
 version=5.10.8
-revision=1
+revision=2
```

Comment:
Probably best to set it through attrition, and let it take effect in the next update.

Reply:
makes sense. done.